src/prebuild/prebuild.csproj

```diff
@@ -18,6 +18,11 @@
       <Visible>false</Visible>
     </GenerateAssembly>
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
It seems like we shouldn't really need to copy this to the output folder - the ThirdPartNotices.txt for the MIEngine isn't actually what you want for the MSI as a whole.